kyuubi-ha/src/main/scala/org/apache/kyuubi/ha/client/ServiceDiscovery.scala

```diff
@@ -24,17 +24,16 @@ import java.util.concurrent.atomic.AtomicBoolean
 import javax.security.auth.login.Configuration
 
 import scala.collection.JavaConverters._
-import scala.util.control.NonFatal
 
 import org.apache.curator.framework.{CuratorFramework, CuratorFrameworkFactory}
-import org.apache.curator.framework.recipes.nodes.PersistentEphemeralNode
```

Comment:
Kindly ping @MyLanPangzi, any update on this PR?

If you have any problem with local debugging, please feel free to let us know.

BTW, we can make a separate PR for this and other import optimization